scripts/install.sh

```diff
@@ -0,0 +1,197 @@
+    elif [ -f "/etc/NIXOS" ]
+    then
+        platform=NixOS
+    else
+        platform=Linux
+    fi
+elif [ "${os}" = "Darwin" ]
+then
+  platform=macOS
+fi
+echo "Found ${platform}."
+
+# Select appropriate package.
+action "Identifying package"
+package=
+if [ "${platform}" = "Debian" ]
+then
+  package="tenzir-linux-static.deb"
+elif [ "${platform}" = "Linux" ]
+then
+  package="tenzir-linux-static.tar.gz"
+elif [ "${platform}" = "NixOS" ]
+then
+  echo "Try Tenzir with our ${bold}flake.nix${normal}:"
+  echo
+  echo "    ${bold}nix run github:tenzir/tenzir/stable${normal}"
+  echo
+  echo "Install Tenzir by adding" \
+    "${bold}github:tenzir/tenzir/stable${normal} to your"
+  echo "flake inputs, or use your preferred method to include third-party"
+  echo "modules on classic NixOS."
+  exit 0
+else
+  echo "We do not offer pre-built packages for ${platform}." \
+       "Your options:"
+  echo
+  echo "  1. Use Docker"
+  echo "  2. Build from source"
+  echo
+  echo "Visit ${bold}https://docs.tenzir.com${normal} for further instructions."
+  exit 1
+fi
+echo "Using ${package}"
+
+# Download package.
+base="https://github.com/tenzir/tenzir/releases/latest/download"
+url="${base}/${package}"
+tmpdir="$(mktemp -u)"
+action "Downloading ${url}"
+if check wget
+then
+  wget -q --show-progress -O "${tmpdir}/${package}" "${url}"
+elif check curl
+then
+  curl --progress-bar -L -o "${tmpdir}/${package}" "${url}"
+else
+  echo "Neither ${bold}wget${normal} nor ${bold}curl${normal}" \
+    "found in \$PATH."
+  exit 1
+fi
+echo "Successfully downloaded ${package}"
+
+# Get platform config.
+prefix=/opt/tenzir
+config="${prefix}/etc/tenzir/plugin/platform.yaml"
+echo
+echo "Unlock the full potential of your Tenzir node as follows:"
+echo
+echo "  1. Go to ${bold}https://app.tenzir.com${normal}"
+echo "  2. Sign up for a free account"
+echo "  3. Configure a node and download a ${bold}platform.yaml${normal} config"
+echo "  4. Move the config to ${bold}${config}${normal}"
+echo
+echo "(You can skip these steps, but then only open source features will be"
+echo "available. Visit ${bold}https://tenzir.com/pricing${normal} for a" \
+  "feature comparison.)"
+echo
+echo "Press ${green}ENTER${normal} to proceed with the installation."
+read -r dummy_
+
+# Check for platform configuration.
+if ! [ -f "${config}" ]
+then
+  open_source=1
+fi
+if [ -n "${open_source}" ]
+then
+  echo "Could not find platform config at ${bold}${config}${normal}."
+  action "Using open source edition"
+fi
+
+# Trigger installation.
+if ! check sudo
+then
+  echo "Could not find ${bold}sudo${normal} in \$PATH."
+  exit 1
+fi
+action "Installing package into ${prefix}"
+if [ "${platform}" = "Debian" ]
+then
+  action "Installing via dpkg"
+  sudo dpkg -i "${tmpdir}/${package}"
+  action "Checking node status"
+  sudo systemctl status tenzir
+elif [ "${platform}" = "Linux" ]
+then
+  action "Unpacking tarball"
+  sudo tar xzf "${tmpdir}/${package}" -C /
+fi
```

Comment:
These two behave quite differently. The former should tell the user that a systemd service was set up for them, and what to do next.

Reply:
Yes, that's unavoidable. We have a guidance section at the end where we should provide specific instructions on what to do next. The text in the middle is not a good place to provide instructions.

I'm already mentioning there that you should start a node if you've used a tarball. I don't think we should mention systemd if it might not be there. We can assume that users can take it from here IMO.

Reply:
The difference is that one starts a node, and the other does not. Can we unify that behavior somehow?

Reply:
I don't think so. It's expected to start a service after `dpkg -i`, but we can't just start a node with service manager that we don't know for the generic case. Instructions are the only way to go here.

Reply:
We'll improve the messaging over time. Let's start with what we have an wait for feedback.